app/views/pages/tutorials/identity-providers/identity-providers.liquid

```diff
@@ -0,0 +1,104 @@
+---
+converter: markdown
+metadata:
+  title: Identity providers
+  description: This guide will help you configure third party identity providers like Facebook login, Auth0 etc.
+slug: tutorials/identity-providers/identity-providers
+searchable: true
+---
+
+This guide will help you understand generic steps of configuring third party identity providers like Facebook login, Auth0 etc.
+Integration allows you to:
+* connect an User from external service with an User in your Instance via email
+* connect currently logged in User with User from an external service
+* register new User with data (name, email) provided by an external service
+
+## Requirements
+To follow the steps in this tutorial, you should be familiar with users. This guide builds on previous tutorials in the **Get Started: Users** section.
+
+* [Users](/tutorials/users/users)
+
+## Steps
+
+All identity provider integrations work the same way and consist of few simple steps:
+
+<div data-autosteps></div>
+
+### Step 1: Create application on Identity Provider website
+You go to third party website to create application
+
+### Step 2: Configure the application
+
+On Identity Provider website  configure application by setting whitelist of valid callback URL, which is https://<YOUR_DOMAIN>/auth/<PROVIDER>/callback
+
+### Step 3: Partner Portal configuration
+
+Go to Partner Portal `https://partners.platform-os.com/instances/<YOUR INSTANCE ID>/edit` and add Client ID, Client Secret and other parameters required by specific provider to configuration -> integrations JSON.
+
+### Step 4: Use /auth/<PROVIDER> endpoint on your Instance
+
+After those steps everything is up and running. You can add for example Log In / Sign Up link on your instance pointing to /auth/<PROVIDER>. It will redirect the User to provider's website and ask for login and password. Then it will redirect the User to /auth/<PROVIDER>/callback and verify the request to ensure user has successfully logged in on provider's site. platformOS will receive token, along with some extra parameters like User's email, first name etc. What happens next depends on the state:
```

Comment:
```suggestion
After those steps, everything is up and running. You can add for example Log In / Sign Up link on your instance pointing to `/auth/<PROVIDER>`. It will redirect the User to the provider's website and ask for login and password. Then it will redirect the User to `/auth/<PROVIDER>`/callback and verify the request to ensure the user has successfully logged in on provider's site. platformOS will receive token, along with some extra parameters like User's email, first name etc. What happens next depends on the state:
```